### ISSUE TITLE
Have elpy-config tell me how to change the RPC Python value

### ISSUE THREAD

**Issue by offby1** [jorgenschaefer/elpy#1472]
So I typed M-x elpy-config RET and saw this:

```
Elpy Configuration

Virtualenv........: None
RPC Python........: 2.7.15 (/apollo/env/SDETools/bin/python)
Interactive Python: python3 (/home/erichan/bin/python3)
Emacs.............: 25.3.50.1
Elpy..............: 1.24.0
Jedi..............: Not found
Rope..............: Not found
```
I would like to change the RPC python to something else, but it's not at all clear how to do so.  In particular, I expanded the `Python` group towards the bottom of the screen, and looked for "RPC", but didn't see it.

Sure I can probably do "C-h a elpy rpc" and see where that leads me, but ... it'd sure be nice if this screen gave me an obvious way to change the values that it's displaying.

# My configuration
## OS
linux

## Result of `(elpy-config)`
```
Elpy Configuration

Virtualenv........: None
RPC Python........: 2.7.15 (/apollo/env/SDETools/bin/python)
Interactive Python: python3 (/home/erichan/bin/python3)
Emacs.............: 25.3.50.1
Elpy..............: 1.24.0
Jedi..............: Not found
Rope..............: Not found
Autopep8..........: Not found
Yapf..............: Not found
Black.............: Not found
Syntax checker....: flake8 (/home/erichan/.local/bin/flake8)

You have not activated a virtual env. While Elpy supports this, it is
often a good idea to work inside a virtual env. You can use M-x
pyvenv-activate or M-x pyvenv-workon to activate a virtual env.

The jedi package is not available. Completion and code navigation will
not work.

[run] easy_install --user jedi

The autopep8 package is not available. Commands using this will not
work.

[run] easy_install --user autopep8

The yapf package is not available. Commands using this will not work.

[run] easy_install --user yapf

The black package is not available. Commands using this will not work.

[run] easy_install --user black

Options

Square brackets indicate buttons; type RET or click mouse-1 on a
button to invoke its action.  Invoke [+] to expand a group, and [-] to
collapse an expanded group.  Invoke the [Group], [Face], and [Option]
buttons below to edit that item in another window.

[+]-- [Group] Elpy
[+]-- [Group] Python
[+]-- [Group] Virtual Environments (Pyvenv)
[+]-- [Group] Completion (Company)
[+]-- [Group] Call Signatures (ElDoc)
[+]-- [Group] Inline Errors (Flymake)
[+]-- [Group] Snippets (YASnippet)
[+]-- [Group] Directory Grep (rgrep)
[+]-- [Group] Search as You Type (ido)
[+]-- [Group] Django extension
[+]-- [Group] Autodoc extension

```

## Elpy configuration in my init.el
```
    (use-package
      elpy
      :config
      ;; I hates it, I does
      (setq elpy-modules (delq 'elpy-module-yasnippet elpy-modules))

      (elpy-enable)
      )
```

**Comment by galaunay** [comment]
It is actually in the `elpy` group.
The name of the option is also mentioned in the doc [here](https://elpy.readthedocs.io/en/latest/concepts.html?highlight=elpy-rpc#option-elpy-rpc-python-command).

But I guess it may be possible to add a button to customize this in `elpy-config`.

**Comment by sten0** [comment]
This seems reasonable, so long as it uses `customize-variable`.  I wonder if the button should only be displayed under specific conditions, because otherwise the action of providing a high-priority custom vars list of buttons might a bug report for whatever var each user has found to be most important for their workflow?

**Comment by galaunay** [comment]
I made the `RPC Python` and `Interactive Python` elements clickable in #1650.
It will ease the customisation of those important variables without adding any clutter to the menu.
Does it sound like a proper solution for you ?

**Comment by galaunay** [comment]
I made the `RPC Python` and `Interactive Python` elements clickable in #1650.
It will ease the customisation of those important variables without adding any clutter to the menu.
Does it sound like a proper solution for you ?

**Comment by galaunay** [comment]
Should be addressed by #1650, 
Please re-open if yo consider it is not :).
